src/test/java/org/openpnp/libuvc4j/test/TestEnumerateCameras.java

```diff
@@ -39,4 +39,13 @@ public static String getString(Pointer<Byte> p) {
 		}
 		return p.getCString();
 	}
+
+	public static String getHexString(int p) {
```

Comment:
I think this is a good change, but can't it be done much simpler with String.format?

Example:
short s = (short) 63190;
System.out.println(String.format("0x%04x", s));
0xf6d6